doc/porting/unreleased.md

```diff
@@ -22,8 +22,14 @@ Otherwise, all elements will be [copied when the array grows](https://stackoverf
 ## ClusterSetSimple and ClusterSetPure
 `ClusterSetSimple` was removed in favor of `ClusterArray<bool>` and `ClusterSetPure` in favor of `ClusterSet<false>` (which does not keep track of its size).
 
+## Graph
+The move constructor and assignment operators of `Graph` are now deleted, which especially means that `NodeArray<Graph>` is no longer possible.
```

Comment:
I would add the following:
... is no longer possible (and neither are `EdgeArray<Graph>`, `FaceArray<Graph>` etc.).